modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenuSmoke.testcase

```diff
@@ -0,0 +1,141 @@
+@component-name = "portal-platform-experience-and-design-system"
+definition {
+
+	property osgi.modules.includes = "accessibility-menu-web";
```

Comment:
Remove this line, not needed to set this anymore since the module is not ignored anymore from builds